applications/TrilinosApplication/python_scripts/PressureMultiLevelSolver.py

```diff
@@ -12,8 +12,7 @@ def MultilevelLinearSolver(tolerance, max_iterations):
     # settings of the ML solver
     MLList = ParameterList()
 
-    default_settings = EpetraDefaultSetter()
-    default_settings.SetDefaults(MLList, "SA")
+    MultiLevelSolver.SetDefaults(MLList, "SA")
```

Comment:
I know these are old, but I have not yet had the time to refactor and integrate it better
Is on my TODO list